lib/pages/setup/route_preview.dart

```diff
@@ -171,3 +172,5 @@ class _RoutePreviewState extends State<RoutePreview> {
     if (_routes.length > 0) switchRoute(_currentRouteIndex);
```

Comment:
We is this method (onMapReady) still in here? I cannot find any place where it is called.
Should probably be deleted before merge?!

Reply:
done